homeworks/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <artifactId>homeworks</artifactId>
-    <packaging>jar</packaging>
+    <packaging>war</packaging>
```

Comment:
Why are you specifying a war if you run the application using embedded tomcat Spring provides you with? Spring runs an app as jar file, so default packaging is good enough.